website/docs/policies/expression.mdx

```diff
@@ -55,18 +55,26 @@ import Objects from "../expressions/_objects.md";
 
 -   `geoip`: GeoIP dictionary. The following fields are available:
 
+    :::info
+    For basic country matching, consider using a [GeoIP Policy](index.md#geoip-policy).
+    :::
+
     -   `continent`: a two character continent code like `NA` (North America) or `OC` (Oceania).
     -   `country`: the two character [ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1) alpha code for the country.
     -   `lat`: the approximate latitude of the location associated with the IP address.
     -   `long`: the approximate longitude of the location associated with the IP address.
     -   `city`: the name of the city. May be empty.
 
     ```python
-    return context["geoip"]["country"] == "US"
+    return context["geoip"]["continent"] == "EU"
```

Comment:
It's possible, e.g. countries in Eurovision famously include Australia :slightly_smiling_face: 

```
    return context["geoip"]["continent"] == "EU" or context["geoip"]["country"] == "AU"
```

but I believe this might be more confusing than just keeping it simple. The params are listed right above, so I don't think "new confusion" arises from changing this.

(I changed this just to suggesting something other that what could be done with a GeoIP policy.)